src/main/java/teammates/common/util/Templates.java

```diff
@@ -298,6 +298,7 @@ public static String populateTemplate(String template, String... values) {
             public static final String CONSTSUM_OPTION_VISIBILITY = "${constSumOptionVisibility}";
             public static final String CONSTSUM_OPTION_POINT = "${constSumOptionPoint}";
             public static final String CONSTSUM_OPTION_VALUE = "${constSumOptionValue}";
+            public static final String MARGIN_LEFT = "${marginLeft}";
```

Comment:
I think here, we prefer if this is done via a class addition rather than a style tag, can we change this solution to using a class addition instead? It's not a good practise to add style tags